CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ## [Unreleased]
 ## [3.0.1] - 2017-07-07
 ### Fixed
+- fix shell call response when running on non en_US locale systems (@stefan-walluhn)
```

Comment:
put this under `[Unreleased]` as we do not modify existing releases. This would be a new patch version (leave all that to us so we can version it when we accept it as there is no guarantee in the order of PRs getting merged.